tls/s2n_tls13.c

```diff
@@ -35,3 +35,9 @@ int s2n_disable_tls13()
     s2n_highest_protocol_version = S2N_TLS12;
     return 0;
 }
+
+/* Returns whether a uint16 iana value is a valid TLS 1.3 cipher suite */
+bool s2n_is_valid_tls13_cipher(const uint8_t version[2]) {
+    /* Valid TLS 1.3 Ciphers are 0x1301, 0x1302, 0x1303, 0x1304, 0x1305 */
```

Comment:
We could add this link to this comment here: https://tools.ietf.org/html/rfc8446#appendix-B.4